lib/db.js

```diff
@@ -86,16 +82,19 @@ exports.reconnect = function(callback) {
 };
 
 /**
- * Closes de db connection
+ * Closes the db connection
  */
 exports.close = function(callback) {
-	exports.addCallback(function() {
-		db.close(true, function (error, result) {
-			isClosed = true;
-			callback (error, result);
-		});
+	if (!db)
+	{
+		return callback(null);
+	}
+	db.close(true, function (error, result) {
+		console.trace('closing');
```

Comment:
I was just debugging an issue with tests, done in commit 9600516cc052617a69b5d550639623149246a49f.